.github/workflows/continuous-integration-workflow.yml

```diff
@@ -36,4 +36,5 @@ jobs:
         python -m pip install --upgrade pip
         pip install -r requirements.txt
     - name: Lint with pylint
-      run: export FLASK_APP_ENV=test; pylint --disable=C0116,E1101 **/*.py
+      run: export FLASK_APP_ENV=test; find . -type f -name "*.py" | xargs p
```

Comment:
Are you able to get a solution working that only uses pylint? To remove the dependency on the 'find' tool? Some solutions might require you to make a `__init__.py` file in /src (and any other subdirectory).